ui/components/Pagination.tsx

```diff
@@ -0,0 +1,118 @@
+import { FormControl, MenuItem, Select } from "@material-ui/core";
+import * as React from "react";
+import styled from "styled-components";
+import Button from "./Button";
+import Flex from "./Flex";
+import Icon, { IconType } from "./Icon";
+import Spacer from "./Spacer";
+import Text from "./Text";
+
+export interface Props {
+  /** CSS MUI Overrides or other styling. */
+  className?: string;
+  /** func for forward one page button */
+  onForward: () => void;
+  /** func for skip to last page button */
+  onSkipForward: () => void;
+  /** func for back one page button */
+  onBack: () => void;
+  /** func for skip to start button */
+  onSkipBack: () => void;
+  /** onChange func for perPage select */
+  onSelect: (value) => void;
```

Comment:
Let's give this `value` a type. Is it a string? Number?

Reply:
it's an event, and it needs to always be an event right? Is there a way to express that in Typescript?

Reply:
Something like `React.SyntheticEvent` or some such:

```suggestion
  onSelect: (ev: React.SyntheticEvent) => void;
```

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/14d95eb0fe90f5e0579c49df136cccdfe89b2855/types/react/index.d.ts#L1167

You can also just leave it blank if you don't expect the user to do anything with the arg.